src/Promitor.Core.Scraping/Scraper.cs

```diff
@@ -83,7 +83,7 @@ public async Task ScrapeAsync(MetricDefinition metricDefinition)
 
                 _logger.LogInformation("Found value '{MetricValue}' for metric '{MetricName}' with aggregation interval '{AggregationInterval}'", foundMetricValue, metricDefinition.Name, aggregationInterval);
 
-                var metricsTimestampFeatureFlag = FeatureFlag.IsActive("METRICSTIMESTAMP", defaultFlagState: true);
```

Comment:
Actually this is intentional for readability

Reply:
addressed